docs/administration/clusterlogforwarder.adoc

```diff
@@ -0,0 +1,59 @@
+= Cluster Log Forwarding
+
+The link:../reference/operator/api.adoc[ClusterLogForwarder] custom resource is the API for specifying the collection and forwarding
+of cluster logs (e.g. application, infrastructure, audit).  The operator supports two modes: Legacy and Multiple-instance.  It operates
+in either of these modes without additional configuration to the operator.  These modes were implemented as part of the
+link:https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/multi-cluster-log-forwarder.md[Multi ClusterLogForwarder]
+feature
+
+== Legacy Mode
+
+The operator is opinionated about the instance namespace and name in this mode and the name of the resulting resources:
```

Comment:
In this mode, the operator is opinionated about the names of ClusterLogging and ClusterLogForwarder objects and the namespace where they reside: